src/Core/Analysis/AnalysisLanguage.cs

```diff
@@ -26,6 +26,7 @@ namespace SonarLint.VisualStudio.Core.Analysis
     public enum AnalysisLanguage
     {
         Javascript,
-        CFamily
+        CFamily,
+        Roslyn
```

Comment:
Seems odd calling `Roslyn` a language. Still, we can always change it later to identify the individual languages if necessary.

Reply:
Yea I wasn't sure, maybe "RoslynFamily"? (like CFamily)

Reply:
That might be slightly better. 

Although returning CFamily also seems odd since we do care whether the language is C or C++, so it seems odd that the language recognizer isn't the thing that calculates it. Still, that's a separate issue.

Reply:
😆 true.

Ok, I renamed to `RoslynFamily` for now.